webapp/src/pages/AuthUserPage/AuthUserPage.tsx

```diff
@@ -54,6 +94,9 @@ export default function AuthUserPage() {
             url={EDIT_USER_URL}
             title="Edit profile"
           />
+          {authUser?.isTwoFactorAuthenticationEnabled
```

Comment:
What do you think about placing this button in the secondaryButton prop of the AvatarPage template instead?

Reply:
Thanks for the suggestion! Done.